cmd/txncleaner/main.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	mgo "gopkg.in/mgo.v2"
+
+	"github.com/juju/txn"
+)
+
+var dbName = flag.String("db", "", "mongo database name (required)")
+var txnsName = flag.String("txns", "txns", "mgo txns collection name")
+var url = flag.String("url", "localhost:27017", "mongo URL")
+
+func main() {
+	flag.Parse()
```

Comment:
my "flag" interaction has been a bit poor, but there are a few things this misses, like supporting SSL connections to mongo (mgopurge has a -ssl=true/false the support doesn't look hard).
It also doesn't seem to allow any sort of user/pass authentication.

As such while it works elsewhere, it actually *won't* work on Juju databases. Ideally we would at least be able to run a 'generic' pruning on Juju dbs as well as other ones.

Also, maybe slightly better help text than just PrintDefaults? Since there isn't any other documentation about what this command really is, why you'd want to use it, when it *isn't* appropriate (the fact that its safe to run live against a database), etc.